### PR TITLE
DAND-6 Bump Facebook SDK

### PR DESCRIPTION
**Pull request by OllaAshour**
**What does this PR do?**

* Update Facebook and Segment Analytics SDK to latest version

**Associated JIRA Tickets:**
https://segmentcontractors.atlassian.net/browse/DAND-6

### CODE REVIEW

File: Segment-Facebook-App-Events.podspec [hunk 1]
```diff
@@ -21,8 +21,8 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0'
-  s.dependency 'FBSDKCoreKit', '~> 4.15'
+  s.dependency 'Analytics'
+  s.dependency 'FBSDKCoreKit'
```
Comment (briemcnally) [inline Segment-Facebook-App-Events.podspec]: Can we add the dependency targets to the .podspec file for 'Analytics' and 'FBSDKCoreKit' here as well?

**Comment by briemcnally** [comment]
Left one comment again about the target dependencies in the podspec file, otherwise looks great! Thanks @OllaAshour!

File: Segment-Facebook-App-Events.podspec [hunk 2]
```diff
@@ -21,8 +21,8 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics', '~> 3.0'
-  s.dependency 'FBSDKCoreKit', '~> 4.15'
+  s.dependency 'Analytics'
```
Comment (brennan) [inline Segment-Facebook-App-Events.podspec]: hey @OllaAshour, echoing brie's comment on the Amplitude PR - can we specify the target version here? thanks!

**Comment by brennan** [comment]
Thanks @OllaAshour. I'll create a new branch to run CI tests, then merge.

**Comment by mericsson** [comment]
Closing due to age.

**Pull request closed without merging**